modules/core/src/smithy4s/codecs/Writer.scala

```diff
@@ -110,8 +110,9 @@ object Writer {
   def noop[Message]: Writer[Message, Any] = (message, _) => message
 
   // format: off
-  implicit def writerEncoderK[Message]: EncoderK[Writer[Message, *], Message => Message] =
-    new EncoderK[Writer[Message, *], Message => Message] {
+  implicit def writerEncoderK[Message]: EncoderK.Aux[Writer[Message, *], Message => Message] =
```

Comment:
**thought:** for public instances like this one, I left `Aux` so that the precise type is still available.